pkg/features/ocp.go

```diff
@@ -0,0 +1,139 @@
+package features
+
+var ocpFeatureMatrix = map[string]Features{
```

Comment:
could do this approach , use yaml 
4.12:
  OC:
    Enabled: true
  BC:
    Enabled: true

4.13:
  inherit: 4.12
  BC:
    DualPort: true

4.14:
  inherit: 4.13
  GM:
    Enabled: true

4.15:
  inherit: 4.14

4.16:
  inherit: 4.15
  BC:
    PTPHA: true
  GM:
    HoldOver: true
    SyncE: true
  LogSeverity: true

Reply:
Do you mean using yaml anchor? 

Or adding a value to features to say grab the previous version, and then combine them at run time?

Reply:
We decided that doing inheritance while preserving the ability to disable features added undesired complexity to the checking code.